### ISSUE TITLE
Processing Information into machine learning

### ISSUE THREAD

**Issue by lrodrz** [lrodrz/eye-of-cthulhu#1]
At this point I can make a list from all of the information parsed from the webpage but now I need to be able to process it to make a predictor. 

This means the next step is to take the data and do the following: 

### **Objectives:**
1. <s>create a list that tells me how many occurrences of each word there are. </s>
   - this has been handled in the webpage object inside of `/tools/webpage.py`
3. take that list and graph it. 
   - this is being handled in `/visualz/viz.ipynb`
5. use that list as an input for ML NN
   - this is the latest update that is being worked on in  `/nerualnet/neural.py`  
7. the output of the list is a prediction of the stock market price of S&P500 (or sports odds?)

****
### **Project Progress/Milestones**:

**Mar-21-2022**: To be able to work with the neural network (NN) it is important that we get some time stamps on the features that we use for the training set. In short, training the data needs a feature set and a output set to put into the regression/regularization model for an estimator meaning there must be a way to relate the data between f(x) = y for x as inputs/features and y as labels.

**Mar-13-2022**: So far we have gotten a good pruning and cleaning application set up with the `tools.webpage.webpage` object. Now we can read through a link and gather the words from it and make a json file with many words in it that can be retrieved and turned into a list. The next thing to do is take that list and make it into an input for the neural network. This will then translate to making predictions using the NN.
